### DISCUSSION TITLE
Allow Two Users to Write in the Same Series

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#15320] [Icebox]
**Is your feature request related to a problem? Please describe.**

An org user recently contacted us because they created a series that alternates authors and they wanted to how they could allow different authors to share the same connected series.

**Describe the solution you'd like**

I think the series writer should be able to turn on "open series" or "closed series" so that others can then write in that series with them.
 
**Describe alternatives you've considered**

This could be too simple of a solution. Perhaps we only want to let people of the same org write in a series together. Or perhaps we want to let series creators create a password for their series which will give others access to write posts in it.

**Additional context**

This issue is different from adding multiple authors to a post - https://github.com/thepracticaldev/dev.to/issues/1299 but it does seem somewhat connected.

**Comment by MarkPieszak** [comment]
Yes I think ideally if this a "Series" within an Organization, it'd be great for multiple authors (inside that organization) to be able to create articles for it.

Thanks for writing this ticket up @michael-tharrington 	!

**Comment by michael-tharrington** [comment]
Something related to this came up recently...

An org admin recently tried to re-credit articles that they'd loaded onto DEV through their admin account, but was having difficulties — the re-credit was unresponsive for some of the articles. The reason: when setting up the articles, the admin adjusted some so that they'd be in a series together. However, when they went to re-credit any articles that were in a series, the system wouldn't let them. And in fact, I had trouble correcting this as an admin. I needed to edit each article, remove the series, credit the correct user, and add the series info back in afterward.

To sum it up, **articles in a series cannot be re-credited to another user.**

**Comment by luxaritas** [comment]
I brought up starting a dev.to org with a project I work on, with the idea of creating one or more series to cover the development stuff we do. Series would be ideal, but it's entirely possible we would want multiple contributors.

**Comment by jbaruch** [comment]
+1 on organizational series. It makes a lot of sense, people in the same organization tend to blog about related topics.

**Comment by JessicaG** [comment]
+1 on organization series, our team does a lot of collaborative work and would be great to credit both authors. 💯

**Comment by JennieOcken** [comment]
There are a lot of requests around making our series functionality more robust.  I want to group them all together here and say we will take them as a set at some point in the future.

[Write description](https://github.com/forem/forem/discussions/15474)
[Edit](https://github.com/forem/forem/discussions/15673)
[Delete](https://github.com/forem/forem/discussions/16434)
[Embed](https://github.com/forem/forem/discussions/16805)
[Multiple Authors](https://github.com/forem/forem/discussions/15320)
[Reorder](https://github.com/forem/forem/discussions/15391)

**Reply by rsjssc** [reply]
Hope this feature will be released in the next version
